components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Registry/src/Deploy/IoTHubDiscovererDeployment.cs

```diff
@@ -104,9 +104,9 @@ public sealed class IoTHubDiscovererDeployment : IHostProcess {
                 // Windows
                 createOptions = @"
                 {
-                    ""HostConfig"": {
-                        ""CapAdd"": [ ""NET_ADMIN"" ]
-                    }
+           //         ""HostConfig"": {
+           //             ""CapAdd"": [ ""NET_ADMIN"" ]
+           //         }
```

Comment:
is this a forgotten comment?

Reply:
This broke overnight and caused windows discovery to fail.  It is unnecessary - I will remove it.